app/assets/stylesheets/application.scss

```diff
@@ -30,3 +31,18 @@
     width: 50%;
    }
  }
+
+ .file-card {
+   background-color: #FF6666;
+   &:hover {
+     box-shadow: 5px 5px 5px grey;
+   }
+ }
+
+ .link-color {
```

Comment:
Make sure the links aren't white on the photo/note show pages since you're using the same class names there for links